lib/iex/lib/iex/info.ex

```diff
@@ -0,0 +1,82 @@
+defprotocol IEx.Info do
+  @spec info(term) :: [{atom, iodata}]
```

Comment:
@fishcakez I'm using a list instead of a map because of ordering (that would be lost with maps). As for the `iodata` not supporting unicode, will fix that when we know this is a good direction.